core/src/it/scala/org/ensime/core/AnalyzerManagerSpec.scala

```diff
@@ -0,0 +1,126 @@
+package org.ensime.core
+
+import org.ensime.api
+import org.ensime.api._
+import org.ensime.fixture.{ EnsimeConfigFixture, IsolatedAnalyzerManagerFixture }
+import org.ensime.model.BasicTypeInfo
+import org.ensime.util.EnsimeSpec
+import org.ensime.util.ensimefile.EnsimeFile
+import org.ensime.util.file._
+class AnalyzerManagerSpec extends EnsimeSpec
+    with IsolatedAnalyzerManagerFixture {
```

Comment:
I'm not sure you need to go to the effort of creating a fixture for this... you should be able to just write mock actors for the underlying actors. That might mean you could make this a unit test instead of an integration test (because you won't need to run the test on a real scala project)